dressForTheWeather/View Controllers/RecommendationViewController.swift

```diff
@@ -9,42 +9,54 @@
 import UIKit
 
 final class RecommendationViewController: UIViewController {
-    
+
     // MARK: - Properties
-    
+
     private var viewModel: RecommendationViewModel!
-    
+    private var backgroundGradientLayer: CAGradientLayer?
+
     // MARK: - IBOutlets
-    
+
     @IBOutlet var temperatureLabel: UILabel!
     @IBOutlet var recommendationLabel: UILabel!
     @IBOutlet var clothingCollectionView: ClothingCollectionView!
-    
+
     // MARK: - Life Cycle
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         viewModel = RecommendationViewModel(delegate: self)
+        setupGradient()
         setBackgroundColor()
     }
-    
+
     // MARK: - Methods
-    
+
     private func updateTempLabel(with temperature: String) {
         temperatureLabel.text = "\(temperature)"
     }
-    
+
     private func updateRecommendationLabel(with recommendations: String) {
         recommendationLabel.text = "\(recommendations)"
     }
     
+    private func setupGradient() {
+        backgroundGradientLayer = CAGradientLayer()
+        backgroundGradientLayer?.frame = view.bounds
+        backgroundGradientLayer?.startPoint = CGPoint(x: 0, y: 0)
+        backgroundGradientLayer?.endPoint = CGPoint(x: 1, y: 1)
+    }
+    
+    private func updateBackgroundColors() {
+        self.backgroundGradientLayer?.removeFromSuperlayer()
+        let colors = self.viewModel.updateBackgroundColors()
+        self.backgroundGradientLayer?.colors = colors
+    }
+
     private func setBackgroundColor() {
```

Comment:
I think this should be named differently as the background colors aren't literally being set here, instead a GradientLayer is being added to the view. Maybe it could be called something like "addBackgroundGradientLayer"?

Reply:
Good call, done in 3fac7db8eb9685c2c9e932523e74c57eea3bca06